extensions/emmet/src/util.ts

```diff
@@ -49,6 +49,28 @@ export function updateEmmetExtensionsPath(forceRefresh: boolean = false) {
 	}
 }
 
+/**
+ * Migrate old configuration(string) for extensionsPath to new type(string[])
+ * https://github.com/microsoft/vscode/issues/117517
+ */
+export function migrateEmmetExtensionsPath() {
+	// Get the detail info of emmet.extensionsPath setting
+	let config = vscode.workspace.getConfiguration().inspect('emmet.extensionsPath');
+
+	// Update Global setting if the value type is string
+	if (typeof config?.globalValue === 'string') {
```

Comment:
This is cool; it only checks the type of the value!
I wonder if we would also need to check if the user explicitly set it to `null`?

Reply:
Oh you got a great point. I think in the old configuration, null is the default value which means that if the user doesn't have any setting to the `emmet.extensionsPath`, it will also be null. Therefore, we might not want to modify the non-exist setting. I will do more tests to see if we can separate the cases between the default `null` and user specified `null`

Reply:
I've checked the configuration return by `vscode.workspace.getConfiguration().inspect()` and it will give us undefined if there is no value given to the `emmet.extensionsPath`, so we can easily separate the different cases!